modules/Head/components/DynamicPreviewBranding.tsx

```diff
@@ -16,15 +15,12 @@ interface Props {
 }
 
 export function DynamicPreviewBranding({ settings }: Props) {
-    const searchParams = JSON.stringify(useSearchParams());
```

Comment:
So, the main problem was that calling `JSON.stringify` was stringifying the result of `useSearchParams` to an empty object even if there were parameters present.